.github/workflows/pr-validation.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         with:
           aws-region: eu-west-1
           role-to-assume: arn:aws:iam::824014778649:role/gh-actions-superwerker-test
```

Comment:
is this role created manually or somewhere in the code? I would prefer having the duration here match the actual session duration of the role so we don't stumble upon another weird error

Reply:
Yeah, I just wanted to make it as short as possible, but you are right, it should probably be 4 hours. Tbh, I don't know where the role is created, it might even be created manually.